.travis.yml

```diff
@@ -1,13 +1,9 @@
 language: python
 python:
   - 2.7
-  - 3.4
-  - 3.5
+  - 3.7
 env:
-  - DJANGO=1.8.14
-  - DJANGO=1.9.9
-  - DJANGO=1.10
-  - DJANGO=1.11
+  - DJANGO=1.11.25
```

Comment:
I wasn't able to get Travis integration working, and don't plan to spend any more time on it for the scope of this ticket. If we want Travis CI integration here, that should be it's own ticket.